### DISCUSSION TITLE
Is it possible to statically generate website from laravel+inertia+vue app

### DISCUSSION THREAD

**Discussion by johnqiuwan** [inertiajs/inertia#1912] [Ideas]
Is it possible to statically generate website from laravel+inertia+vue app ？ Hybrid some static page could be serve the request much faster and user friendly. Does any plan inertia will support on ssg generate, which is same as what nextjs is providing. Thank you

**Comment by derrickreimer** [comment]
This falls outside the scope of what Inertia is designed to do, but you could look at using something like this to generate static versions of pages: https://github.com/spatie/laravel-export

**Reply by johnqiuwan** [reply]
Thank you for you reply, and I will look into that.
